src/serializer.ts

```diff
@@ -3,10 +3,18 @@ import { Transaction } from "../generated/transaction_pb";
 import { FiatAmount } from "../generated/fiat_amount_pb";
 import { XRPAmount } from "../generated/xrp_amount_pb";
 import { Currency } from "../generated/currency_pb";
+import Utils from "./utils";
 
 /* Allow `any` since this class doing progressive conversion of protocol buffers to JSON. */
 /* eslint-disable @typescript-eslint/no-explicit-any */
 
+interface PaymentJSON {
+  Amount: object | string;
+  Destination: string;
+  DestinationTag?: number | undefined;
```

Comment:
This allows `DestinationTag` to be omitted, or if it is provided, it can be a `number` or `undefined`. I think it would make more sense for it to be one of these instead:
```
DestinationTag?: number;
```
or
```
DestinationTag: number | undefined;
```

Reply:
Done